x-pack/plugins/cases/server/client/test_utils.ts

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
As this file contain mocks and not utility functions maybe it is better to rename the file to `user_profiles.mock.ts`